src/functions/hskip.js

```diff
@@ -0,0 +1,65 @@
+//@flow
+// Horizontal spacing commands
+
+import defineFunction from "../defineFunction";
+import buildCommon from "../buildCommon";
+import mathMLTree from "../mathMLTree";
+import { calculateSize } from "../units";
+import ParseError from "../ParseError";
+
+// TODO: \hskip and \mskip should support plus and minus in lengths
```

Comment:
Good call splitting this out, tbh I'm not sure how to do this, at least for our current HTML layout strategy.

Reply:
Yeah, definitely for another time.  Supporting `fil`/`fill`/`filll` units in `plus` at least should be possible using Flexbox.  That'd be pretty useful -- but for another issue.  I've opened #990 to track.